tools/create-hybrid-image

```diff
@@ -150,9 +160,19 @@ function sanity_check
 		echo "" 1>&2
 	fi
 
+	if (( $(pfexec bash -c 'echo $EUID') != 0 )); then
```

Comment:
is `pfexec /bin/id -u` equivalent, or not portable enough?

Reply:
That should be portable to everything that can run `pfexec`.